dll/cpl/input/lang/cs-CZ.rc

```diff
@@ -10,10 +13,10 @@ STYLE DS_SHELLFONT | WS_CHILD | WS_DISABLED | WS_CAPTION
 CAPTION "Nastavení"
 FONT 8, "MS Shell Dlg"
 BEGIN
-    GROUPBOX "Default input &language", -1, 5, 5, 246, 55
-    LTEXT "Select one of the installed input languages to use when your computer starts.", -1, 15, 15, 230, 23
+    GROUPBOX "Výchozí vstupní &jazyk", -1, 5, 5, 246, 55
+    LTEXT "Vyberte jeden z nainstalovaných jazyků, který se bude používat po spuštění Vašeho počítače.", -1, 15, 15, 230, 23
     COMBOBOX IDC_DEFAULT_LANGUAGE, 15, 40, 230, 300, CBS_HASSTRINGS | CBS_AUTOHSCROLL | CBS_DROPDOWNLIST | WS_VSCROLL | WS_TABSTOP
-    GROUPBOX "&Installed services", -1, 5, 65, 246, 125
+    GROUPBOX "Na&instalované služby", -1, 5, 65, 246, 125
```

Comment:
Picture please that proved the **MS** cs-CZ does use a small &i here. They do typically recommend to avoid that.

Reply:
@JoachimHenze It's actually on the "s".
![Screenshot of Windows XP' language control panel in Czech](https://konoha.cz/files/jp/003.png)